### ISSUE TITLE
AI Basics: What are 

### ISSUE THREAD

**Issue by pdehaan** [mozilla/ai-guide#13]
### Sub-section title

AI Basics

### Please describe your issue

https://ai-guide.future.mozilla.org/content/ai-basics/#what-are-the-pros-cons-of-using-an-llm something is wrong with that slug. IIRC we looked at it before and the & wasn’t encoding so it is probably `pros--cons` instead of `pros-cons`.

```ini
a href="/content/ai-basics/#what-are-the-pros-cons-of-using-an-llm"
h4 id="what-are-the-pros--cons-of-using-an-llm"
```

### Describe the solution you'd like to see

Change the `href` to use `pros--cons` so it finds the anchor and fixes the sidebar nav.

**Comment by Offirmo** [comment]
Correct. It's here https://github.com/mozilla/ai-guide/blob/3b779129574e947e3f8ee6af133155910c08d077/templates/content/ai-basics/_sidebar.html#L10
